rest_framework/templates/rest_framework/horizontal/select_multiple.html

```diff
@@ -11,7 +11,7 @@
   {% endif %}
 
   <div class="col-sm-10">
-    <select multiple {{ field.choices|yesno:",disabled" }} class="form-control" name="{{ field.name }}">
```

Comment:
how can we determine this won't create any regression?

Reply:
From my side I will include the modified template in our current projects. So far this seems to have resolved the issue of forms not loading within 30 seconds that the application server is willing to wait on DRF request processing and the end user getting a gateway timeout.

Reply:
Another way to reason about this: as long as field.choices are present (field queryset is non-empty), then the select element is not assigned the "disabled" html attribute. Theoretically speaking it should not matter whether a multi-select field without any choices is disabled or enabled - there isn't anything to select.

Is anyone aware of some corner case situation that we should be aware of?

Reply:
is it possible to add test coverage for this?